resources/contents/blog/en/review-2018.md

```diff
@@ -0,0 +1,38 @@
+I started that way. In the beginning of 2018 I got in touch with some people that I hadn't spoken to for a long time (like for years). I also started publishing some work to open source.
+
+## From Ghent with love
+
+But during March/April there was a big chance coming towards me. In a sub-clause of my review I mentioned that I wanted to move to a Dutch speaking area and work there for a while. And by the end of March I got that opportunity: I got the opportunity to work at Teamleader in the beautiful city of Ghent. And without wasting too much time I took this chance and moved to Gent in May. [I even wrote about it here](https://rathes.me/blog/en/from-ghent-with-love)
+That change destroyed my other plans. Instead of getting back in touch with people that I hadn't talked too much I met and still meet new people. For the first time in my life I'm living longer than 6 months in another country than Germany and I am pretty sure that I'm not going to move back. My plans to continue learning Japanese have been delayed, instead I'm trying to survive in an environment with a lot of Flemish accents and the amount of work (as an employee) definitely increased compared to last year.
+But how do I feel about it? Even after now almost eight months I still think that taking the chance here was one of the best decisions I've ever made in my entire life. It might not help me on my life-time goal, but moving here and working in such a company with daily learnings gives me so many more opportunities in the mid-term. Opportunities that I wouldn't have gotten in Germany.
+
+## 2019 is all about learning (and teaching)
+
+For the upcoming year I do not want to set any big goal. I want to focus that year entirely on learning. Learning to become a better developer, learning to become a speaker, learning to become a better leader, learning to become a better human.
+In addition I want people to learn from me. I'm probably not the smartest man on earth (and I will never be), but I think that I made some experiences in life already that might be interesting for other people. I want to spead the learnings I made and help others. Not sure how I'm going to do that, but I will figure out.
+Another thing that I want to accomplish next year is getting something done that I postponed for too long: Releasing my app to learn Tamil. I'm working on it for way too long but in the last days I made some real progress. In the upcomig days I will be able to release the first landing page that will keep you updated about the first grammar book that I'm going to release and based on that I will publish an iOS app.
+
+## Let's get things done
+
+A piece of advice that I can give you: It doesn't matter how much you plan and envision, if you don't execute it will stay a dream. So whatever you think of you should do, go out and make it happen.
```

Comment:
```suggestion
A piece of advice that I can give you: It doesn't matter how much you plan and envision if you don't execute it will stay a dream. So whatever you think of you should do, go out and make it happen.
```